added favicons on all pages and changed a title

unnat1654 opened the pull request:
Fixed #194 

Added favicons to all pages after cropping the logo of ChatTutor given in the readme file.

adrianariton reviewed (changes requested):
Would it be possible to overlay a black rectangle over the C (that is still present in the logo)? Maybe use an online photoeditor or photoshop

unnat1654 commented:
> Would it be possible to overlay a black rectangle over the C (that is still present in the logo)? Maybe use an online photoeditor or photoshop

i really apologize for the long delay. i do not understand what you mean by black rectangle

adrianariton commented:
Basically to make the C inside the logo disappear

unnat1654 commented:
I changed the favicon. Please check if it fits your requirements.
I tried inverting the color of the logo and it looked slightly better, i have added that image as well, if you want then i will change the favicon to that image.

adrianariton commented:
Yup, this is perfect! I will merge this today!
